neurom/features/tests/test_get_features.py

```diff
@@ -478,7 +478,7 @@ def npts(neurite):
     def vol(neurite):
         return neurite.volume
 
-    features.register_neurite_feature('foo', npts)
+    features.register_neurite_feature('foo2', npts)
```

Comment:
It would be fine if you replace `foo` with `foo2` and `bar` with `bar2` also in the other occurrences in the same function `test_register_neurite_feature_pop`.
But unit tests should not modify the global state, because the results of other tests could be potentially affected by the order of tests, so a better solution is to use `patch.dict` when a dict is used as global variable.
Adding `@patch.dict(FEATURES['NEURITEFEATURES'])` to the functions that call `register_neurite_feature` should do the trick (because it's the nested dict that is modified, so I'm not sure if `@patch.dict(FEATURES)` would be enough).

Reply:
I see your point and I agree however I don't manage to make it work. I have tried as a decorator, as a context manager, using directly the variable `NEURITEFEATURES` or the string `'neurom.features.NEURITEFEATURES'` but to no avail. So I've kept my poor man's method...